components/molecules/PendingBenefitDetails.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import VerticalStepper from '../atoms/VerticalStepper'
+import en from '../../locales/en'
+import fr from '../../locales/fr'
+
+const PendingBenefitCard = (props) => {
+  const t = props.locale === 'en' ? en : fr
+
+  return (
+    <div className="grid grid-cols-1 divide-y divide-grey-500 font-display my-4">
+      <section>
+        <div className="sm:flex sm:justify-around sm:grid sm:grid-cols-7 sm:gap-4 sm:mx-auto sm:mt-4">
+          <div
+            id="claimUpdates"
+            className="col-span-2 sm:col-span-3 lg:col-span-2 sm:w-60 ml-7 mb-6 sm:mb-3"
+          >
+            <VerticalStepper
+              benefitStatus={props.benefit.status}
+              lastUpdates={props.benefit.lastUpdates}
+              locale={props.locale}
+            />
+          </div>
+
+          <div
+            id="details"
+            className="col-span-5 sm:col-span-4 lg:col-span-5 sm:grid mr-5"
```

Comment:
`className="col-span-5 sm:col-span-4 lg:col-span-5 sm:grid mr-5"`

Don't need col-span-5, twice here. Something like below will accomplish the same thing:
`className="col-span-4 mr-5 sm:grid sm:col-span-5"`

Reply:
![mobile-screens](https://user-images.githubusercontent.com/51453299/151883601-ed7f1a2b-0a8b-4b08-bf9e-a0ac1630da64.jpg)
https://tailwindcss.com/docs/responsive-design

Reply:
Ok sounds good